tests/unit/utils/test_fs.py

```diff
@@ -164,3 +164,49 @@ def test_remove(repo_dir):
 
     remove(path_info)
     assert not os.path.isfile(path_info.fspath)
+
+
+@pytest.mark.parametrize(
+    "parent",
+    [
+        (os.path.join("path", "to", "")),
+        (os.path.join("path", "to")),
+        (os.path.join("path", "")),
+        (os.path.join("path")),
+    ],
+)
```

Comment:
Please, simply inline this.

Reply:
It doesn't make much harm here aside from some bloat, but under other circumstances, e.g. if fixture is used, it might slow down tests.